### PR TITLE
chore(docs): Added badges in README.md file

### PR DESCRIPTION
**Pull request by sumitkharche**
Added below badges:
- Github release
- Go Report Card
- GitHub License
- Docker pulls count

### CODE REVIEW

**Comment by itaysk** [comment]
Hi and thanks for the contribution. As per our contribution guidelines and the Hacktoberfest rules, can you please create an Issue and associate this PR with the issue? Thanks

**Comment by sumitkharche** [comment]
#239

**Comment by sumitkharche** [comment]
> Hi and thanks for the contribution. As per our contribution guidelines and the Hacktoberfest rules, can you please create an Issue and associate this PR with the issue? Thanks

Thanks for the review @itaysk  I have created an issue and linked this PR as well.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ![Tracee Logo](images/tracee.png)
+
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/releases)
+[![Go Report Card](https://goreportcard.com/badge/github.com/aquasecurity/tracee)](https://goreportcard.com/report/github.com/aquasecurity/tracee)
+[![GitHub](https://img.shields.io/github/license/aquasecurity/tracee)](https://github.com/aquasecurity/tracee/blob/master/LICENSE)
```
Comment (itaysk) [inline README.md:5]: just rename the alt here from GitHub to License and I'll approve

**Review by itaysk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sumitkharche) [reply README.md]: Done

**Review by itaysk — APPROVED** [review]

**Comment by kichloo** [comment]
@itaysk I am a beginner in open source and don't know much technologies till now. Are there any easy/documentation issues which I can work on?

**Comment by itaysk** [comment]
@kichloo if you are new to open source and looking for guidance, please see this guide: https://hacktoberfest.digitalocean.com/details#beginners
and if you have further general questions about contributing to open source you can try the Hacktoberfest Discord forum: https://discord.com/invite/hacktoberfest. If we have easy issues we would label them as `good-first-issue` but we don't have in this repo. In the future, please don't post general questions in this thread, If you have questions about Tracee specifically you can open a new "Discussion" in the Discussions tab above.

**Pull request merged**